neo/rawio/intanrawio.py

```diff
@@ -1278,25 +1268,27 @@ def create_one_file_per_channel_dict_rhd(dirname):
     file_names = dirname.glob("**/*.dat")
     files = [file for file in file_names if file.is_file()]
     raw_file_paths_dict = {}
-    for raw_index, prefix in enumerate(possible_raw_file_prefixes_rhd):
-        raw_file_paths_dict[raw_index] = [file for file in files if prefix in file.name]
-
-    raw_file_paths_dict[6] = [Path(dirname / "time.dat")]
-
+    for stream_name, file_prefix in stream_name_to_file_prefix_rhd.items():
```

Comment:
I like adding issues as provenance but I think the best place for them is in the regression test. That said, I don't see why we can't have it on both places for this example.

As a side comment, I think it would be even better if I could also put the channel files on a dict but that would require a bigger refactor than I had the time to do.